src/Admin/SiteHealth.php

```diff
@@ -534,6 +534,16 @@ public function get_page_cache_detail( $use_previous_result = false ) {
 		];
 	}
 
+	/**
+	 * Callback for {@see get_page_cache_headers()}.
+	 *
+	 * @param string $header_value Header value.
+	 * @return bool Whether header contains the value "hit"
+	 */
+	private static function cache_hit_callback( string $header_value ): bool {
```

Comment:
This was introduced in bd2ee42bf5c035ad1d3868d6f40400d6af13d99d apparently to deal with a PHPCS bug, and not due to any fault of the code?

Reply:
Correct